config/rbac/rbac_role.yaml

```diff
@@ -37,9 +37,9 @@ rules:
   - patch
   - delete
 - apiGroups:
-  - kfserving.kubeflow.org
+  - serving.kubeflow.org
   resources:
-  - services/status
+  - kfservices/status
   verbs:
```

Comment:
So the docs are referring to ' kfserving' resource, and then we flip between kfservices and serving.....would it make sense to be consistent and prepend everything with kf?

Reply:
kfservice is one of the resources in the group serving.kubeflow.org. Another resource might be kfgraph in serving.kubeflow.org. Make sense?